beacon-chain/powchain/service.go

```diff
@@ -329,8 +358,14 @@ func (w *Web3Service) run(done <-chan struct{}) {
 			log.Debug("ETH1.0 chain service context closed, exiting goroutine")
 			return
 		case w.runError = <-headSub.Err():
-			log.Debugf("Unsubscribed to head events, exiting goroutine: %v", w.runError)
-			return
+			if w.runError.Error() == "EOF" {
+				log.Infof("Got EOF: restarting rpc client")
```

Comment:
This should be a WARN, please use `log.Warn(...`

Reply:
thanks